src/encoded/tests/test_types_submission_center.py

```diff
@@ -1,3 +1,3 @@
 def test_types_submission_center(testapp, test_submission_center):
     """ Tests we can post/get a consortium item """
-    testapp.get(f'/submission-center/{test_submission_center["name"]}')
```

Comment:
A random place to mention this, but ... do we want to have version numbers on this new protocol in case we later have to change it?